docs/UserGuide.md

```diff
@@ -9,331 +9,424 @@ Interface** (CLI) while still providing an easy way to visualize all events thro
 Events are displayed in chronological order.  
 </div>
 
-[[Back to top](#index)]
-
+[[Back to top](#table-of-contents)]
 
-<h3 id="upcoming">Upcoming Events</h3>
+<h3 id="upcoming-events-panel">Upcoming Events Panel</h3>
 ![upcoming-events.png](images/upcoming-events.png)
-Displays ongoing (in case you have not start on it yet) and upcoming events you have. 
 
-<div markdown="span" class="alert alert-info">
+This feature displays both ongoing (if you have not started on them yet) and upcoming events that you have.
+
+<div markdown="span" class="alert alert-success">
 :bulb: **Tip:**
-While the default is one upcoming event,
-you can use the `next` command to change the number of upcoming events shown.
+While the default is one upcoming event, you can use the `next` command to change the number of upcoming events shown.
 </div>
 
-[[Back to top](#index)]
-
+[[Back to top](#table-of-contents)]
 
 <h3 id="calendar">Calendar</h3>
 ![calendar.png](images/calendar.png)
-Provides an overview of all the event that you have.
 
-Functionality:  
-1. Events that are added will be shown in its respective calendar box.
-2. On click of `now` button will jump you directly back to the current date.
-3. On click of any calendar date boxes that have events will display the events in event list.
-4. `find` command will highlight the calendar boxes that has the event.
+This feature gives you a comprehensive summary of all the events that you have for a particular month.
+
+Functionality:
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:**
+Newly added events will appear in their corresponding calendar boxes.  
+The find command will highlight the calendar boxes corresponding to the events that match your search criteria.
+</div>
 
-[[Back to top](#index)]
+<div markdown="span" class="alert alert-success">
+:bulb: **Tip:**
+Clicking on the `now` button will take you directly back to the current date.  
+Clicking on any calendar date box that has events will display those events in the event list.
+</div>
 
+[[Back to top](#table-of-contents)]
 
+<h2 id="summary">Summary</h2>
+
+<h3 id="prefix-summary">Prefix summary</h2>
+
+| Prefix | Representation |
+|:-------|:---------------|
+| n/     | Name           |
+| d/     | Date           |
+| s/     | Start time     |
+| e/     | End time       |
+| every/ | Every          |
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** Name is alphanumeric only.
+</div>
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** Dates should follow a `YYYY-MM-DD` format.
+</div>
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** Time should be in 24-hour `HH:mm` format, ranging from `00:00` to `23:59`. Leading zero are required. 
+</div>
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** Recurring factor is given by `day/week/month`.
+</div>
```

Comment:
Repeated information (already provided in the respective command). Duplicating information makes this document harder to maintain. (May also information overload the user when reading the individual commands.)